aom/src/main/java/com/nedap/archie/adl14/ADL14NodeIDConverter.java

```diff
@@ -153,15 +136,18 @@ private List<String> findUnnecessaryCodes(CObject cObject, Map<String, Archetype
         return result;
     }
 
+    /**
+     * Replace old id's in term definition with the new codes
```

Comment:
One line 156-161, it seems that there is a bug, because the newTerm variable is never used, so I expect that needed to be used on line 161 instead just 'term'. But I don't want to change any functionality in this PR. Might look at this later.